test/integration/scheduler/util.go

```diff
@@ -162,11 +162,12 @@ func createNodes(cs clientset.Interface, prefix string, wrapper *st.NodeWrapper,
 // createAndWaitForNodesInCache calls createNodes(), and wait for the created
 // nodes to be present in scheduler cache.
 func createAndWaitForNodesInCache(testCtx *testutils.TestContext, prefix string, wrapper *st.NodeWrapper, numNodes int) ([]*v1.Node, error) {
+	nodeList, _ := testCtx.ClientSet.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
```

Comment:
why not `sched.SchedulerCache.NodeCount`, assuming that you can only use this function, but not `createNodes` directly?

Reply:
SG. updated.